### PR TITLE
TORE paper added

### PR DESCRIPTION
**Pull request by bald6354**
I'd like to add our TORE paper to the wiki page. It is an event representation that we tested against 3d human pose estimation using DHP19 dataset. Does it make sense to add a new category for human pose estimation? I think there are at least 3 papers now.

We also tested against intensity image reconstruction, event denoising, and classification. Since this builds on our prior work in denoising I put the main entry there and added links for the other areas.

It is currently published to arxiv. Thanks for maintaining this site. It is incredibly useful!!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -706,6 +706,8 @@ arXiv, 2020.
 - <a name="GantierCadena21tip"></a>Gantier Cadena, P. R., Qian, Y., Wang, C., Yang, M.,  
 *[SPADE-E2VID: Spatially-Adaptive Denormalization for Event-Based Video Reconstruction](https://doi.org/10.1109/TIP.2021.3052070)*,  
 IEEE Trans. Image Processing, 30:2488-2500, 2021. [Project page](https://github.com/RodrigoGantier/SPADE_E2VID)
+- [Baldwin R.W., Ruixu L., Almatrafi M., Asari V., Hirakawa K.](#Baldwin21arxiv),
```
Comment (guillermogb) [inline README.md:709]: We use only one last name for "pointers" (2nd categories). See Barua et al, example.

File: README.md [hunk 2]
```diff
@@ -1696,6 +1702,9 @@ IEEE Trans. Multimedia (TMM), 2020.
 - <a name="Guo20arxiv"></a>Guo, S., Kang, Z., Wang, L., Zhang, L., Chen, X., Li, S., Xu, W.,  
 *[A Noise Filter for Dynamic Vision Sensors based on Global Space and Time Information](https://arxiv.org/pdf/2004.04079)*,  
 arXiv, 2020.
+- <a name="Baldwin21arxiv"></a>Baldwin R.W., Ruixu L., Almatrafi M., Asari V., Hirakawa K.,
```
Comment (guillermogb) [inline README.md:1705]: See the name formatting of other entries... for consistency there should be commas separating last names from given names.

File: README.md [hunk 3]
```diff
@@ -1696,6 +1702,9 @@ IEEE Trans. Multimedia (TMM), 2020.
 - <a name="Guo20arxiv"></a>Guo, S., Kang, Z., Wang, L., Zhang, L., Chen, X., Li, S., Xu, W.,  
 *[A Noise Filter for Dynamic Vision Sensors based on Global Space and Time Information](https://arxiv.org/pdf/2004.04079)*,  
 arXiv, 2020.
+- <a name="Baldwin21arxiv"></a>Baldwin R.W., Ruixu L., Almatrafi M., Asari V., Hirakawa K.,  
+*[Time-Ordered Recent Event (TORE) Volumes for Event Cameras](https://arxiv.org/abs/2103.06108)*,  
+arXiv, 2021. [PDF](https://arxiv.org/abs/2103.06108), [Code](https://github.com/bald6354/tore_volumes).
```
Comment (guillermogb) [inline README.md:1707]: No need to add the additional [PDF] link after the title until the official publication venue, for example ieee, occupies the link in the title.

**Review by guillermogb — COMMENTED** [review]
Hi,

Yes, it makes sense to create a new category for human pose estimation now that there are "enough" papers.

Since the TORE paper seems to be about representations, similar to Gehrig et al ICCV 2019, End-to-End Learning of Representations for Asynchronous Event-Based Data, wouldn't it make sense to use the same principal category as Gehrig et al ICCV 2019? We know that representations can be used for many applications.

File: README.md (hunk 1 above)
Comment (bald6354) [reply README.md]: ok, making the change

File: README.md (hunk 2 above)
Comment (bald6354) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (bald6354) [reply README.md]: ok

File: README.md [hunk 4]
```diff
@@ -706,8 +706,7 @@ arXiv, 2020.
 - <a name="GantierCadena21tip"></a>Gantier Cadena, P. R., Qian, Y., Wang, C., Yang, M.,  
 *[SPADE-E2VID: Spatially-Adaptive Denormalization for Event-Based Video Reconstruction](https://doi.org/10.1109/TIP.2021.3052070)*,  
 IEEE Trans. Image Processing, 30:2488-2500, 2021. [Project page](https://github.com/RodrigoGantier/SPADE_E2VID)
-- [Baldwin R.W., Ruixu L., Almatrafi M., Asari V., Hirakawa K.](#Baldwin21arxiv),  
-*Time-Ordered Recent Event (TORE) Volumes for Event Cameras.*
+- [Baldwin et al., Arxiv 2021](#Baldwin21arxiv). Time-Ordered Recent Event (TORE) Volumes for Event Cameras.
```
Comment (guillermogb) [inline README.md:709]: Arxiv -> arXiv 
everywhere please :P

**Review by guillermogb — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (bald6354) [reply README.md:709]: haha, sorry

**Comment by bald6354** [comment]
I added pointers to the papers under the category 3D Human Pose Estimation. There was an existing category for 3D Hand Estimation, but it was missing from the main top index. I corrected this. It may make sense to add additional papers to these categories. If I find some that make sense I will submit another pull request.

File: README.md [hunk 5]
```diff
@@ -1843,7 +1851,18 @@ Robotics: Science and Systems (RSS), 2020. [PDF](http://www.roboticsproceedings.
 IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2020.
 
 
-## 3D hand pose estimation
+<a name="pose_estimation"></a>
+## Pose Estimation
```
Comment (guillermogb) [inline README.md:1855]: Thanks. Could we call it "Object Pose Estimation" ?
I may be biased by my background, but when I read "pose estimation" I think of "(camera) pose estimation"

File: README.md [hunk 6]
```diff
@@ -47,6 +47,9 @@
     - [Control](#control)
     - [Space Applications](#space)
     - [Tactile Sensing Applications](#tactile_sensing)
+    - [Pose Estimation](#pose_estimation)
+        - [3D Human Pose Estimation](#3d_human_pose_estimation)
+        - [3D Hand Pose Estimation](#3d_hand_pose_estimation)
```
Comment (guillermogb) [inline README.md:52]: thanks for fixing the missing entry in the toc

**Review by guillermogb — APPROVED** [review]

**Comment by guillermogb** [comment]
Thanks, Baldwin (I assume). I will add Event-GAN to the list and align like Bardow's pointer (paper title starts a new line -- I should have done this for Barua's).

**Comment by bald6354** [comment]
Thank you again! Yes, Wes Baldwin. We meet a couple of times pre-covid :smile: I'm looking forward to the workshop at CVPR this year! Will you have another survey paper?

**Comment by guillermogb** [comment]
Is there a need for another survey paper? What is missing? (besides the new deep learning papers?)

**Comment by bald6354** [comment]
Nothing specific. Just lots of new sensors, datasets, algorithms, applications, etc. Honestly, I use this site to track the latest developments. In the survey paper, you outlined issues and limitations that needed to be addressed. I'd like to know if you think the community has made progress in these areas? I feel like the newer generation of sensors will allow us to tackle new problems, but the volume of data means I can no longer work on event camera projects on my tablet PC (i.e. even sparse data volume is an issue). What are the coming challenges? Like this site, it's an opportunity to keep research focused on real problems. Just my two cents. :smile:

**Comment by guillermogb** [comment]
Thanks for your input. I think it is going to be difficult to keep research focused the more people get to work on the sensors. Each researcher or group has its own agenda and opinion on what "real problems" means. 
- Regarding the new sensors, their higher data throughout implied by their higher spatial resolution and possibly sensitivity is indeed a big challenge. We need a computational platform (hardware) that scales and evidence of this. The community is aware that it is missing.
- Even before going into higher resolution, I think we need benchmarks to be able to compare algorithms for a given task and make steady progress (to know what works and what does not work, etc.). We tried to highlight this need in the survey paper. New datasets are coming, indeed. Meaningful evaluation protocols and metrics are also needed.
- Finally, it is not all about datasets. In robotics we like to play with the camera (change its parameters, lenses, etc.)

Not a new survey paper, but I tried to put some effort into making it more educational:
https://sites.google.com/view/guillermogallego/teaching/event-based-robot-vision
Hope it helps.

**Comment by bald6354** [comment]
Wow! This material is great! Can I share this with some new grad students?

**Comment by guillermogb** [comment]
Thanks. Sure, feel free to share it.
It was listed in here, too: https://github.com/uzh-rpg/event-based_vision_resources#teaching

**Comment by guillermogb** [comment]
This EU project is compiling links, too:
https://neurotechai.eu/resources/references/

**Pull request merged**